src/hotspot/os/bsd/os_perf_bsd.cpp

```diff
@@ -36,8 +36,20 @@
   #include <mach/task_info.h>
 #else
   #ifndef __NetBSD__
+    #ifdef __FreeBSD__
+      /*
+       * Older versions of FreeBSD accidentally include machine/frame.h from
+       * sys/user.h header. Disable this bad behavior, because a
+       * 'non-standard' structure 'frame' conflict with an internal structure
+       * with the same name.
+       */
+      #define _MACHINE_PCB_H_
+    #endif
     #include <sys/user.h>
   #endif
+#if !defined(__arm__)
+  #include <sys/user.h>
+#endif
```

Comment:
Haven't we already included this two lines above?  Maybe this is a leftover from trying to avoid the frame definition conflict in other ways.

Reply:
some leftover, I'm struggling with git